src/InfoBoxes/Content/Type.hpp

```diff
@@ -144,8 +144,10 @@ namespace InfoBoxFactory
     e_NbrSat, /* Number of used Sat by GPS module */
 
     e_ActiveRadio, /* Active Radio Frequency */
-
     e_StandbyRadio, /* Standby Radio Frequency */
+
+    e_TransponderCode, /* Transponder code */
```

Comment:
Adding new enum choices in the middle breaks all existing configurations. Add at the end (right before `e_NUM_TYPES`).

Reply:
done